developer_manual/design/layoutcomponents.rst

```diff
@@ -6,15 +6,16 @@ All Nextcloud apps are built using individual reusable components for consistenc
 Navigation
 ----------
 
+`Vue component for the app navigation <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers/NcAppNavigation?id=ncappnavigation-1>`_.
+
+`Penpot Navigation Items <https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=3f784c86-6c27-80c6-8002-6ab157b6bd27&section=interactions&index=8&share-id=88d87192-d205-8007-8002-8d8d21a0dcca>`_
```

Comment:
This link doesn’t seem to work – probably best to use the same one as in https://github.com/nextcloud/documentation/pull/10431 ?

(Also not sure why – Penpot links seem to be buggy?)

Reply:
done

Reply:
@szaimen and probably same in https://github.com/nextcloud/documentation/pull/10397 as well – thank you! :)